themes/digital.gov/layouts/events/card-event-past.html

```diff
@@ -8,9 +8,9 @@
     </div>
     {{- else -}}
     <div class="non-youtube-event-card grid-col-12 tablet:grid-col-4 tablet:order-last grid-row">
-      <div aria-role="img" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
-        <svg xmlns="http://www.w3.org/2000/svg" role="img" height="150" width="150" viewBox="0 0 25 25">
-          <title>Non-Youtube Event</title>
+      <div aria-role="img" alt="" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
+        <svg xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="calendar-title" height="150" width="150" viewBox="0 0 25 25">
+          <title id="calendar-title">Calendar Event</title>
```

Comment:
Not necessary because we've hidden it, but we can leave.

Reply:
I have left this since it is a best practice but not used in this case.